dags/parquet_import.py

```diff
@@ -0,0 +1,61 @@
+import os
+from datetime import datetime
+from airflow.models import DAG
+from airflow.operators.python import PythonOperator
+from airflow.providers.postgres.operators.postgres import PostgresOperator
+
+def get_latest_month_year():
+    currentMonth = datetime.now().month - 1
+    currentYear = datetime.now().year
+    if currentMonth == 12:
+        currentYear = currentYear - 1
+    if currentMonth < 10:
+        currentMonth = f'0{currentMonth}'
+    return f'_{currentYear}-{currentMonth}'
+
+
+def download_from_s3_csv(ti,prefix_file:str,bucket_name: str) -> str:
+    latest_month_year = ti.xcom_pull(task_ids=['get_latest_month_year'])[0]
+    file_name_path = f'{local_path}{prefix_file}{latest_month_year}.csv'
+    s3_path = f's3://{bucket_name}/{prefix_file}{latest_month_year}.parquet'
+    os.system(f'parquet-tools csv "{s3_path}" > "{file_name_path}"')
```

Comment:
It might be good to use the [BashOperator](https://airflow.apache.org/docs/apache-airflow/stable/howto/operator/bash.html) here instead of `os.system`. I'm not sure what would happen with the `os.system` approach if an error occurs, i.e. if it would show the error message. I haven't used the `BashOperator` so far, but would assume it is more robust for errors, retries, and such things.